t/t7519-status-fsmonitor.sh

```diff
@@ -106,6 +106,8 @@ EOF
 
 # test that "update-index --fsmonitor-valid" sets the fsmonitor valid bit
 test_expect_success 'update-index --fsmonitor-valid" sets the fsmonitor valid bit' '
+	write_script .git/hooks/fsmonitor-test<<-\EOF &&
+	EOF
```

Comment:
Isn't the integration script (i.e. `fsmonitor-test`) initialized only two test cases further down?

I guess this change is necessary because the fsmonitor was not exercised in this test case without this current patch. If that is the case, I would prefer to initialize `fsmonitor_test` as an empty script in the `setup` test case already.